exercises/anagram/src/test/java/AnagramTest.java

```diff
@@ -1,9 +1,10 @@
-import static org.assertj.core.api.Assertions.assertThat;
+import org.junit.Ignore;
+import org.junit.Test;
 
 import java.util.Arrays;
 import java.util.List;
-import org.junit.Test;
-import org.junit.Ignore;
+
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
OMG, you're going to kill me — this is a long PR; thank you for your patience.

Looks like we have a freeloader, here.